code/game/machinery/doors/airlock.dm

```diff
@@ -2251,66 +2402,173 @@
 // CentCom Airlocks
 
 /obj/machinery/door/airlock/centcom //Use grunge as a station side version, as these have special effects related to them via phobias and such.
-	icon = 'icons/obj/doors/airlocks/centcom/centcom.dmi'
-	overlays_file = 'icons/obj/doors/airlocks/centcom/overlays.dmi'
+	icon = 'icons/obj/doors/airlocks/tall/centcom.dmi'
+	overlays_file = 'icons/obj/doors/airlocks/tall/overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_centcom
 	normal_integrity = 1000
 	security_level = 6
 	explosion_block = 2
+	greyscale_config = null
+	greyscale_colors = null
 
 /obj/machinery/door/airlock/grunge
-	icon = 'icons/obj/doors/airlocks/centcom/centcom.dmi'
-	overlays_file = 'icons/obj/doors/airlocks/centcom/overlays.dmi'
+	icon = 'icons/obj/doors/airlocks/tall/centcom.dmi'
+	overlays_file = 'icons/obj/doors/airlocks/tall/overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_grunge
 
 
 // Vault Airlocks
 
 /obj/machinery/door/airlock/vault
 	name = "vault door"
-	icon = 'icons/obj/doors/airlocks/vault/vault.dmi'
+	icon = 'icons/obj/doors/airlocks/tall/vault/vault.dmi'
 	overlays_file = 'icons/obj/doors/airlocks/vault/overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_vault
 	explosion_block = 2
 	normal_integrity = 400 // reverse engieneerd: 400 * 1.5 (sec lvl 6) = 600 = original
 	security_level = 6
+	greyscale_config = null
+	greyscale_colors = null
 
+/obj/machinery/door/airlock/vault/animation_length(animation)
+	switch(animation)
+		if(DOOR_OPENING_ANIMATION)
+			return 1.9 SECONDS
+		if(DOOR_CLOSING_ANIMATION)
+			return 1.9 SECONDS
+		if(DOOR_DENY_ANIMATION)
+			return 0.3 SECONDS
+
+/obj/machinery/door/airlock/vault/animation_segment_delay(animation)
+	switch(animation)
+		if(AIRLOCK_OPENING_TRANSPARENT)
+			return 1.3 SECONDS
+		if(AIRLOCK_OPENING_PASSABLE)
+			return 1.8 SECONDS
+		if(AIRLOCK_OPENING_FINISHED)
+			return 1.9 SECONDS
+		if(AIRLOCK_CLOSING_UNPASSABLE)
+			return 0.3 SECONDS
+		if(AIRLOCK_CLOSING_OPAQUE)
+			return 0.6 SECONDS
+		if(AIRLOCK_CLOSING_FINISHED)
+			return 1.9 SECONDS
 
 // Hatch Airlocks
 
 /obj/machinery/door/airlock/hatch
 	name = "airtight hatch"
-	icon = 'icons/obj/doors/airlocks/hatch/centcom.dmi'
-	overlays_file = 'icons/obj/doors/airlocks/hatch/overlays.dmi'
-	note_overlay_file = 'icons/obj/doors/airlocks/hatch/overlays.dmi'
+	icon = 'icons/obj/doors/airlocks/tall/hatch/centcom.dmi'
+	//overlays_file = 'icons/obj/doors/airlocks/hatch/overlays.dmi'
+	//note_overlay_file = 'icons/obj/doors/airlocks/hatch/overlays.dmi'
```

Comment:
we do not yet have overlays for airlocks, it's one of the bounties that's currently unfinished, so yes. I don't want to use the entirely wrong lights if I can help it since they would look even worse